atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/knowledge/KnownConcludedTransactionsStoreTest.java

```diff
@@ -60,6 +63,34 @@ public void tracksDistinctRanges() {
                                 .add(Range.closedOpen(1L, 100L))
                                 .add(Range.closedOpen(150L, 200L))
                                 .build())
+                        .minimumTimestamp(0L)
                         .build());
     }
+
```

Comment:
think we can add one more `rangesBelowMinTsAreIgnored`

Reply:
Yes! This will be added in the next PR, as this will throw currently (as there are is no intersection)